src/Microsoft.DocAsCode.Build.Engine/SystemMetadataGenerator.cs

```diff
@@ -48,24 +48,26 @@ public SystemMetadata Generate(InternalManifestItem item)
             var fileWithoutWorkingFolder = file.RemoveWorkingFolder();
             attrs.Path = fileWithoutWorkingFolder;
 
-            // 1. Root Toc is always in the top directory of output folder
-            var rootToc = _toc.FirstOrDefault();
-            if (rootToc != null)
+            // 1. Root Toc is specified by RootTocKey, or by default in the top directory of output folder
+            if (!string.IsNullOrEmpty(_context.RootTocPath))
             {
-                var rootTocPath = rootToc.File.RemoveWorkingFolder();
-                if (rootTocPath.SubdirectoryCount == 0)
+                if (((TypeForwardedToRelativePath)_context.RootTocPath).IsFromWorkingFolder())
```

Comment:
>((TypeForwardedToRelativePath)_context.RootTocPath).IsFromWorkingFolder()) [](start = 20, length = 74)

by default paths in docfx.json are from working folder, does user need to specify `~` for roottocpath?